### PR TITLE
A new stub for sql union all

### PR DESCRIPTION
**Pull request by nemesis1311**
Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

**Comment by Bouncey** [comment]
Deploy preview ready!

Built with commit ebacc8594096769a3c7cd363fe967812f20c6f42

https://deploy-preview-7959--guide.netlify.com

File: src/pages/sql/sql-union-all-operator/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| Hammer Inc.                    |
+| PS Plastics                    |
++--------------------------------+
+
+```
+The use of UNION ALL or UNION depends upon your requirements. If you need duplicate or redundant rows from your query UNION ALL would be a better approach for writing your query.
+
+As with all of these SQL things there is MUCH MORE to them than what's in this introductory guide.  
+
+I hope this at least gives you enough to get started.
```
Comment (amandeepmittal) [inline src/pages/sql/sql-union-all-operator/index.md]: Please remove this line.  And also if you could provide the link to the manual for the link below.

Comment (amandeepmittal) [reply src/pages/sql/sql-union-all-operator/index.md]: @nemesis1311 Please address these changes.

**Comment by domwakeling** [comment]
Marked as stale and closed due to inactivity. :disappointed:

Please make appropriate changes if you have the time :+1:

**Pull request closed without merging**